addons/sprite-properties/addon.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "Collapsing sprite properties",
+  "description": "Hides the sprite properties panel by default, like in Scratch 2.0. Use the info button on the currently selected sprite or double-click a sprite to show the properties panel again. To re-hide it, use the collapse button in the properties panel or double-click a sprite.",
+  "tags": ["editor"],
+  "credits": [
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://github.com/lisa-wolfgang/"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["projects"]
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "customCssVariables": [
+    {
+      "name": "transitionDuration",
+      "value": {
+        "type": "map",
+        "source": {
+          "type": "settingValue",
+          "settingId": "transitionDuration"
+        },
+        "options": {
+          "none": "0s",
+          "short": "0.15s",
+          "default": "0.25s",
+          "long": "0.5s"
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "name": "Collapse panel on load",
```

Comment:
I think we could use a better term than "on load" here

Reply:
onion-skinning uses `Enable by default` for pretty much the same setting meaning.
We could use that or maybe `Collapse panel by default`